Gems/ROS2ScriptIntegration/Code/Platform/Mac/PAL_mac.cmake

```diff
@@ -0,0 +1,4 @@
+
+set(PAL_TRAIT_ROS2SCRIPTINTEGRATION_SUPPORTED TRUE)
+set(PAL_TRAIT_ROS2SCRIPTINTEGRATION_TEST_SUPPORTED FALSE)
+set(PAL_TRAIT_ROS2SCRIPTINTEGRATION_EDITOR_TEST_SUPPORTED FALSE)
```

Comment:
For Windows, Android, Mac, iOS Gem does not make much sense.
I would consider simplifying build process to only support Linux.

Reply:
MacOS and Windows are Tier3 ROS 2 supported platforms.